get-artifacts/README.md

```diff
@@ -0,0 +1,54 @@
+# Get Artifacts #
+
+A downloader for artifacts from multiple prior workflow_runs, or a single run by workflow_run_id.
+
+Artifacts are always uploaded as zips, correspondingly this action allows you unzip and then delete the original downloads with the "decompress" flags.
+
+If a ```${{ inputs.target_dir }}/<workflow_run_id>``` folder already exist no attempt will be made to download the artifacts for that run.
```

Comment:
needs some grammar fixes. Also are we going to preserve the downloaded artifacts between runs? It's not clear to me how this caching is going to work in practice.

Reply:
there's a github action that can cache files between workflow runs, we already do this for cargo crates.   this would do a catchup if any runs failed/were skipped.

Reply:
fixed up and linked to the cache documentation.